### DISCUSSION TITLE
Why is there a file size upload limit of 2GB?

### DISCUSSION THREAD

**Discussion by Shieldowskyy** [tris203/disftp#21] [Q&A]
On Disbox in the browser, as far as I know, there is no limit on the file upload size. Here the server does not allow uploading files larger than 2GB. Will this change in the future?

**Comment by tris203** [comment]
Interesting @Shieldowskyy 

I don't know if I tried a file that big

I don't remember intentionally putting a limit on it

What error do you get?

**Reply by Shieldowskyy** [reply]
![image](https://github.com/tris203/disftp/assets/32707076/5286842a-5441-4cf7-8718-9357410e07be)

**Reply by tris203** [reply]
Buffer size in node

I would have to rewrite with streams to support larger files

Not impossible just would require thinking about

Was this downloading or uploading?

**Reply by tris203** [reply]
I think it will only be uploads
